### ISSUE TITLE
Using RajawaliVR with current Rajawali main branch

### ISSUE THREAD

**Issue by yakobom** [Rajawali/RajawaliVR#5]
Hi,
I was not able to get this working - seems like the current Rajawali dev env has changed (Android Studio support?) so I cannot compile it, the required imports cannot be found...
Any ideas?

Thanks,
yakobom

**Comment by flankechen** [comment]
@MasDennis 
Agree, Rajawali master branch is developing quite fast these days, while VR and AR depends on out of day codes.
I found that even RajawaliActivity is removed. 
hopefully, VR and AR branch would catch up the main branch.

**Comment by MasDennis** [comment]
AR is up to date but VR isn't yet. We are working on this.

**Comment by yakobom** [comment]
OK, I will be waiting for that.
Thank you very much.

yakobom

**Comment by MasDennis** [comment]
Hi @yakobom I managed to do it already. Please check the latest update and tell me if you have any problems :beers:

**Comment by yakobom** [comment]
Thanks a lot for the notification!
I will check it out and let you know.

yakobom

**Comment by yakobom** [comment]
Hi,
Just to let you knw - I have left it for now, spent to much time trying to build it, perhaps something is wrong with my private configuration.

**Comment by SebastienANTOINE** [comment]
Cannot able to build it. Seems broken

**Comment by MasDennis** [comment]
@SebastienANTOINE can you be a bit more specific? Do you get any messages?

**Comment by thebradbain** [comment]
@MasDennis I'm not able to build it either. I'll open up a separate issue.

**Comment by RamIndani** [comment]
I was facing same issue so I opened ticket https://github.com/Rajawali/RajawaliVR/issues/14
I pulled branch instead of master and it worked for me.

**Comment by drbago** [comment]
The description of #14 sounds similar to this issue and provides enough details to troubleshoot.

Unfortunately, this thread does not have enough information to troubleshoot the issue.  The only tangible details I see are the following:
> I cannot compile it, the required imports cannot be found...

and 

> [not] able to build it.

Given this information, it seems to me that this is probably a duplicate issue of #14 -- or perhaps a completely different issue with an invalid description.  If folks do not give some more clear details (like stack traces or error messages), then I recommend @MasDennis close this ticket and label it as invalid.

**Comment by jferrer-vhm** [comment]
With https://github.com/Rajawali/RajawaliVR/pull/18, I am able to easily build from the Gradle CLI.

However, I am having trouble building the project from Android Studio using the .idea project folders in the git repo.

**Steps to reproduce**
1. Check out a fresh version of the project.
2. Open the /RajawaliVRExample/.idea project in Android Studio

**Expected results**
Upon opening the project, just select the SDK path for the project and Run/Debug Configuration should be usable.

**Actual results**
IDEA errors, missing modules, defunct run configurations, no Android facets

It seems that the repo is missing a couple of IDEA .iml module files that make the .idea projects unusable without significant tweaking.  Eventually, I was able to get it running, but I clobbered my changes and am now having a hard time.  If the problem is missing modules, @MasDennis , would you mind committing those missing .iml files?
